udsoncan/j2534.py

```diff
@@ -243,12 +256,18 @@ def PassThruReadVersion(self, DeviceID):
 
         return Error_ID(hex(result)), pFirmwareVersion, pDllVersion, pApiVersion
 
-    def PassThruIoctl(self, Handle, IoctlID, ioctlInput=None, ioctlOutput=None):
+    def PassThruGetLastError(self):
+        pErrorDescription = (ctypes.c_char * 80)()
+        result = dllPassThruGetLastError(pErrorDescription)
 
+        return Error_ID(hex(result)), str(pErrorDescription.value)
+
+    def PassThruIoctl(self, Handle, IoctlID, ioctlInput=None, ioctlOutput=None):
         if ioctlInput is None:
             pInput = POINTER(c_ulong)()
         else:
-            pInput = c_ulong(ioctlInput.value)
+            # pInput = c_ulong(ioctlInput.value)

```

Comment:
This comment becouse incompatible with passing Ioctl_Flags.TX_IOCTL_SET_DLL_DEBUG_FLAGS